projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -3456,7 +3456,7 @@ public NodeRoleSpecifier getNodeRoleSpecifier(boolean inferred) {
       inferredRoles = true;
       nodeRolesPath = settings.getInferredNodeRolesPath();
       if (!Files.exists(nodeRolesPath)) {
-        result = new NodeRoleSpecifier();
+        return new NodeRoleSpecifier();
```

Comment:
Before @millstein 's recent change, this code was just a `return` statement. I believe this is the desired behavior; otherwise the assignment on this line was unused (it got overwritten at L3462 below).